src/inmanta/module.py

```diff
@@ -3041,10 +3037,7 @@ def __init__(
     ) -> None:
         self._is_editable_install = is_editable_install
         self._version: Optional[version.Version] = installed_version
-        try:
-            super(ModuleV2, self).__init__(project, path)
-        except InvalidMetadata as e:
-            raise InvalidModuleException(f"The module found at {path} is not a valid V2 module") from e
+        super(ModuleV2, self).__init__(project, path)
```

Comment:
This change makes me a bit nervous so I triggered the slow tests on your branch. I wonder if we could make `InvalidMetadata` inherit from `InvalidModuleException` and get the best of both worlds? We could then drop the `except InvalidMetadata` in moduletool as well.

Reply:
@sanderr so the slowbuild suceeded, do you still want me to try to make InvalidMetadata inherit from InvalidModuleException? (I haven't look at it at all yet )

Reply:
Ok i'll give it a try :)